docs/pages/release_notes.md

```diff
@@ -10,13 +10,7 @@ The PMD team is pleased to announce PMD {{ site.pmd.version }}.
 
 This is a {{ site.pmd.release_type }} release.
 
-### Table Of Contents
-
-* [New and noteworthy](#new-and-noteworthy)
-    *   [New Rules](#new-rules)
-* [Fixed Issues](#fixed-issues)
-* [API Changes](#api-changes)
-* [External Contributions](#external-contributions)
+{% tocmaker is_release_notes_processor %}
```

Comment:
what's this `is_release_notes_processor`?

Reply:
The toc is generated only if that variable is true, so when pushing to github or sourceforge, but not when jekyll renders the site (in which case Jekyll already generates a toc).